### PR TITLE
Issue #1093 Display last verified date in service detail

### PR DESCRIPTION
**Pull request by niko-pg**

### CODE REVIEW

File: src/validation/search/__tests__/to_human_service_data.tests.ts [hunk 1]
```diff
@@ -233,6 +240,7 @@ describe('Adapting search result to service data', () => {
                 service_name: aString(),
                 service_id: aString(),
                 service_description: aString(),
+                last_verified_date: aDate().toISOString(),
```
Comment (rasmus-storjohann-PG) [inline src/validation/search/__tests__/to_human_service_data.tests.ts:243]: Looks like `aDate()` should maybe simply return the ISO string...

File: src/components/services/service_detail_component.tsx [hunk 2]
```diff
@@ -124,7 +124,13 @@ const ServiceContactDetails = (props: Props): JSX.Element => {
...
                 analyticsLinkPressed={props.analyticsLinkPressed}
             />
+<<<<<<< Updated upstream
             <ContentVerificationComponent verificationDate={'N/A'} />
         </View>
+=======
+            <DividerComponent />
+            <ContentVerificationComponent verificationDate={props.service.lastVerifiedDate} />
+        </>
+>>>>>>> Stashed changes
```
Comment (rasmus-storjohann-PG) [inline src/components/services/service_detail_component.tsx]: Please resolve conflicts before committing, our aim is that all commits compile and pass all unit tests. In this PR the simplest solution is to squash this commit with the following one.

File: src/validation/search/__tests__/to_human_service_data.tests.ts [hunk 3]
```diff
@@ -1,10 +1,6 @@
 // tslint:disable:no-expression-statement
 import { toHumanServiceData } from '../to_human_service_data';
-<<<<<<< Updated upstream
-import { aString, aNumber } from '../../../application/helpers/random_test_values';
-=======
-import { aDate, aString, aNumber } from '../../../helpers/random_test_values';
->>>>>>> Stashed changes
+import { aDate, aString, aNumber } from '../../../application/helpers/random_test_values';
```
Comment (rasmus-storjohann-PG) [inline src/validation/search/__tests__/to_human_service_data.tests.ts:3]: Please fix this one too

File: src/validation/services/__tests__/to_human_service_data.test.ts [hunk 4]
```diff
@@ -60,6 +60,7 @@ describe('Adapting server service object to client service object', () => {
             email: serverServiceObject.service.organization_email,
             organizationName: serverServiceObject.service.organization_name,
             bookmarked: false,
+            lastVerifiedDate: serverServiceObject.service.last_verified_date,
```
Comment (rasmus-storjohann-PG) [inline src/validation/services/__tests__/to_human_service_data.test.ts:63]: Commit message says you added a test, but you didn't

File: src/components/services/service_detail_component.tsx (hunk 2 above)
Comment (nikojohnarellano) [reply src/components/services/service_detail_component.tsx]: Definitely. Will squash the commit

**Pull request merged**